src/Compilers/CSharp/Test/Emit/Emit/EndToEndTests.cs

```diff
@@ -27,16 +27,16 @@ public void OverflowOnFluentCall()
 
             switch (IntPtr.Size * 8) {
                 case 32 when isDebug:
-                    numberFluentCalls = 290;
+                    numberFluentCalls = 510;
```

Comment:
I haven't looked into that. I'm assuming the JIT just doesn't optimize and there may be some additional overhead (temps for assertions and such).
Do you think this needs investigation?